c34904525.lua

```diff
@@ -1,8 +1,8 @@
 --雷風魔神－ゲート・ガーディアン
 function c34904525.initial_effect(c)
+	aux.AddCodeList(c,25955164,62340868)
```

Comment:
`aux.AddFusionProcMix` and `aux.AddFusionProcMixRep` is calling `aux.AddCodeList` by default.